src/components/card/card.tsx

```diff
@@ -52,17 +59,22 @@ export const Card: React.FC<CardProps> = ({
 
   return (
     <div
-      className={clsx("mykn-card", {
-        "mykn-card--full-height": fullHeight,
-        [`mykn-card--direction-${direction}`]: direction,
-        [`mykn-card--justify-${justify}`]: justify,
-      })}
+      className={clsx(
+        "mykn-card",
+        {
+          "mykn-card--full-height": fullHeight,
+          "mykn-card--border": border,
+          [`mykn-card--direction-${direction}`]: direction,
+          [`mykn-card--justify-${justify}`]: justify,
+        },
+        className,
+      )}
       {...props}
     >
       {(actions.length || title) && (
         <div className="mykn-card__header">
           <Body fullHeight>
-            {typeof title === "string" ? <H3>{title}</H3> : title}
+            {typeof title === "string" ? <H2>{title}</H2> : title}
```

Comment:
This was changed earlier in b40742fa, maybe we could either:

- Change the implementation to provide JSX.
- Add a titleAs prop?